code/client/src/pages/Create.jsx

```diff
@@ -106,6 +107,13 @@ const Create = () => {
       message.error('Cannot deploy wallet. Error: root is not set.')
       return
     }
+
+    // Ensure valid address for both 0x and one1 formats
+    const validLastResortAddress = util.validateAddress(lastResortAddress)
+    if (!validLastResortAddress) {
+      return
```

Comment:
It would be quite hard to keep tracking error handling if the message prompt is absorbed in above function and its name does not suggest it does handle errors.